SwiftWisdom/Core/StandardLibrary/String/String+Indexing.swift

```diff
@@ -53,6 +53,8 @@ extension String {
         }
     }
 
+    // TODO: Rename to ip_removeFirst()?
+    // In std library drop() returns a new String rather than mutating in place
     public mutating func ip_dropFirst() {
```

Comment:
Should we make this unavailable then? And point users to the swift functions `dropFirst` and `removeFirst`?

Reply:
That would be my thinking. `removeFirst` in the standard library crashes if called on an empty, so I could imagine there might still be a use case for this function along the lines of the safely indexing subscripts. But then it would more logically be called `ip_safelyRemoveFirst()` and return a `Character?` with `@discardable result`.

Reply:
I like it